bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/console/LinkConsoleCommandExtension.java

```diff
@@ -89,7 +90,8 @@ public void execute(String[] args, Console console) {
                         ChannelUID channelUID = new ChannelUID(args[2]);
                         addChannelLink(console, itemName, channelUID);
                     } else {
-                        console.println("Specify item name and channel UID to link: link <itemName> <channelUID>");
+                        console.println(
+                                "Specify item name and channel UID to link: addChannelLink <itemName> <channelUID>");
```

Comment:
`addChannelLink` comes from a time when there were also `ThingLinks`. Since they do not exist anymore, I wonder if we shouldn't rather simplify the command to be simply `link` and `unlink` again, just as the help text says. Wdyt?

Reply:
Yes, I agree. We are using it in the new UI too and it should be the new openHAB naming. I will change it.